static/_headers

```diff
@@ -1,4 +0,0 @@
-/*
```

Comment:
And, just so you know, it's not in the theme yet. I didn't think to add it until I saw yours here.